toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1286,19 +1301,52 @@ func runPostInstallScripts(installChroot *safechroot.Chroot, config configuratio
 	return
 }
 
+func setGrubCfgAdditional(grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
```

Comment:
matching suggestion for nit

```suggestion
func setGrubCfgAdditionalCmdLine(grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
```